components/footer.tsx

```diff
@@ -79,11 +81,16 @@ function FooterItem({ title, links }) {
     <div>
       <div className={styles["column-header"]}>{title}</div>
       <ul>
-        {links.map(({ name, href }, idx) => (
-          <li key={idx}>
-            <a href={href} target="_blank">{name}</a>
-          </li>
-        ))}
+        {links.map(({ name, href }, idx) => {
+          return name === "Newsletter" ?
+            <li key={idx}>
+              <a onClick={() => ml_webform_4255780('show')}>{name}</a>
+            </li>
+          :
```

Comment:
I think it would be better to remove the conditional out of the map body, and have it as a separate `<li>`. Since "Newsletter" is first in the list, we can statically declare that it should be rendered first.
```suggestion
        <li>
            <a onClick={() => ml_webform_4255780('show')}>Newsletter</a>
        </li>
        {links.map(({ name, href }, idx) => {
```

Reply:
We use the Footer Item component for each column, meaning we would be rendering the Newsletter link at the top of each column if we did it this way. I'm going to revert this change and merge the PR, but we can revise the approach in a a new PR if necessary.